.github/workflows/build-nuget-package.yaml

```diff
@@ -1,49 +1,30 @@
-name: Publish MAUI Binding Package
+name: Build NuGet Package
 
 on:
-  release:
-    types: [published]
+  workflow_call:
+    outputs:
+      version:
+        description: "Package version"
+        value: ${{ jobs.build-nuget-package.outputs.version }}
   workflow_dispatch:
 
 jobs:
-  publish:
-    name: Publish to GitHub Packages
+  build-nuget-package:
+    name: Build and Pack NuGet Package
     runs-on: macos-latest
+    outputs:
+      version: ${{ steps.get_version.outputs.version }}
 
     steps:
       - name: Checkout Repository
         uses: actions/checkout@v4
 
-      - name: Setup Java
-        uses: actions/setup-java@v4
-        with:
-          java-version: 17
-          distribution: temurin
-
       - name: Import iOS Certificate
         uses: apple-actions/import-codesign-certs@v3
         with:
           p12-file-base64: ${{ secrets.IOS_P12_BASE64 }}
           p12-password: ${{ secrets.IOS_P12_PASSWORD }}
 
-      - name: Setup .NET 9
-        uses: actions/setup-dotnet@v4
-        with:
-          dotnet-version: '9.0.x'
-          source-url: https://nuget.pkg.github.com/emartech/index.json
-        env:
-          NUGET_AUTH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
-
-      - name: Install .NET MAUI
-        run: |
-          dotnet workload install maui
-
-          # Install compatible iOS workload version for Xcode 16.4
-          dotnet workload install ios --version 9.0.100
-
-      - name: Setup Android SDK
-        uses: android-actions/setup-android@v3
-
       - name: Install Objective-Sharpie and Xamarin.iOS
```

Comment:
we may remove sharpie and xamarin.ios installation step.
it is disabled for now in the [csproj](https://github.com/emartech/maui-binding-for-sap-emarsys/blob/main/ios/Emarsys.iOS.Binding/Emarsys.iOS.Binding.csproj#L25) .
it is needed only for development, to help generate the [ApiDefinition.cs](https://github.com/emartech/maui-binding-for-sap-emarsys/blob/main/ios/Emarsys.iOS.Binding/ApiDefinition.cs) .